CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ set(RULES_PREFIX   "60"                             CACHE PATH "The numeric pref
 
 set(TBT_CXXFLAGS ${CXX_FLAGS} -Wall -Wextra)
 
+option(use-std-filesystem "use-std-filesystem" OFF)
```

Comment:
Maybe better to plan ahead for C++ library with full C++17 compatibility where FS is under `std::` namespace (not `std::experimental::`)? It means having 3 options: boost (the default for now), filesystem TS (`std::experimental`) and std-filesystem (`std`).